src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlCachedBuffer.cs

```diff
@@ -58,7 +56,7 @@ internal static bool TryCreate(SqlMetaDataPriv metadata, TdsParser parser, TdsPa
                     break;
```

Comment:
NIT: Even though it's not mandatory for single-line if-statements, it could prevent misunderstanding, and I suggest using braces even for single-line statements like this.
```suggestion
                {
                    break;
                }
```